great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -155,6 +158,7 @@ def rules(self) -> Optional[List[Rule]]:
         numeric_rule: Rule = self._build_numeric_rule()
         datetime_rule: Rule = self._build_datetime_rule()
         categorical_rule: Rule = self._build_categorical_rule()
+        text_rule: Rule = self._build_text_rule()
```

Comment:
@cdkini very minor -- stylistic -- let us put the text rule before categorical and after numeric and date/time rule.